src/main/webapp/translate_sentiment.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
I thought we were keeping the sentiment translator on the home page? Otherwise LGTM!!